kategory/src/main/kotlin/kategory/instances/EitherMonad.kt

```diff
@@ -1,6 +1,6 @@
 package kategory
 
-class EitherMonad<L> : Monad<EitherF<L>> {
+open class EitherMonad<L> : Monad<EitherF<L>> {
```

Comment:
I'd rather it used delegation rather than inheritance like in here: https://github.com/kategory/kategory/blob/master/kategory/src/main/kotlin/kategory/instances/Composed.kt#L39